### PR TITLE
tests(websocket): websocket integration tests

### LINKED ISSUE #1241: Integration test - JSON-RPC Websockets
Websocket might have a lot of tech dept since it wasn't worked on for a long time. We need to improve reliability through a test suite.

Create a websocket test suite for the Ethereum JSON-RPC that is able to subscribe to the different events, blocks, logs, etc:

### PR DESCRIPTION
**Pull request by adisaran64**
Closes: #1241 

## Description

added skeleton for websocket tests, cannot implement with current version of web3.py
along with #1342 , closes #1241 
______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: tests/integration_tests/test_websockets.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from .expected_constants import (
+    EXPECTED_FEE_HISTORY,
+    EXPECTED_GET_PROOF,
+    EXPECTED_GET_STORAGE_AT,
+    EXPECTED_GET_TRANSACTION,
+    EXPECTED_GET_TRANSACTION_RECEIPT,
+)
+from .utils import (
```
Comment (danburck) [inline tests/integration_tests/test_websockets.py:8]: @adisaran64 can you fix the flake8 warnings?

**Review by danburck — COMMENTED** [review]
@adisaran64 can you add some tests for some the rpc calls, e.g. `getBlockNumber` (https://ethereum.org/en/developers/tutorials/using-websockets/) ? Or is the batch request blocking you to do this? 

It would helpful to add one example of what you can query with websockets here: https://docs.evmos.org/developers/clients.html#ethereum-websocket

**Comment by fedekunze** [comment]
@adisaran64 what's the status of this PR?

**Comment by adisaran64** [comment]
@danburck modified the `test_types.py` file to test the ethermint websocket along with the RPC, so that all rpc calls currently tested are also tested on the websocket. will open a PR to add that example. fixed flake issues as well, needed to ignore some clashing rulesets (long lines vs. line breaks)

**Comment by hanchon** [comment]
LGTM, tested it locally and it's working fine. The most useful part is the fixture with the normal rpc and the websocket (`ethermint_rpc_ws` )

**Review by fedekunze — APPROVED** [review]

**Pull request merged**